librosa/beat.py

```diff
@@ -546,7 +548,9 @@ def plp(
     return util.normalize(pulse, axis=-1)
 
 
```

Comment:
For now I focused on the public interface only, because this was already so much in itself

Reply:
Sure, I was mainly asking because there were a part or two where private helpers also got some type annotation (in the lpc function, IIRC).  But that's fine.